src/App.jsx

```diff
@@ -3,10 +3,12 @@ import { Dashboard } from "./components/dashboard/dashboard";
 import { SignIn } from "./components/signIn/signIn";
 import { Register } from "./components/register/register.jsx";
 import { List } from "./components/list/list";
+import Navbar from "./components/Navbar/Navbar.jsx";
 
 function App() {
   return (
     <div className="App">
+      <Navbar/>
```

Comment:
navbar is covering the top of the first card and the top of the list page.

can you make the navbar color "darkpink" so it catches more attention? Palepink is mostly for the background

(hex codes are in one of the other comments and in discord dms)